cnx/src/text.rs

```diff
@@ -132,6 +132,12 @@ pub struct Attributes {
     pub padding: Padding,
 }
 
+pub struct PagerAttributes {
+    pub active_attr: Attributes,
```

Comment:
Can you add documentation on each of the struct field on what they are ? Eg: Active attributes is applied to the current active workspace. 

For `non_empty_attr`, we can specifiy that it is a `workspace` which has no windows in it currently.

Reply:
I added a doc comment to every field of the struct.